src/components/HyperlinkingWrapper.js

```diff
@@ -129,6 +144,8 @@ export default class HyperlinkingWrapper extends Component {
 
     renderTooltip() {
         const { current, isLinkInvalid, currentSelection } = this.state;
+        const format = this.quill.getFormat();
+        const linkValue = (typeof format.link === 'string') ? format.link : null;
```

Comment:
Why do we need to check `format.link` type?
I would also set `linkValue` as a state field in `onSelection`.

Reply:
done